atlp-rwanda/e-comm-team-techsmith-bn
# #184752074 Set up an empty Express Boilerplate with dotenv

## What does this PR do?
This PR aims at setting up express and server project
## Description of Task to be completed?

1. Set up an empty Node.js - express application. 
2. By this We mean a full directory with all folders that you think developer will need. this might include but not limited to 
   src/ controllers, models, routes helpers,validations, etc
3.Test
4.  feel free to contact the rest of the team and get ideas from them because this is the first skeleton of the application.
5. Install and setup dotenv for the API to manage environment variables. Ensure packages like sequelize and passport among the rest can fetch data about the current environment correctly via dotenv. Add documentation on how to utilize dotenv in the API.

6. Set the server and make sure it is running

## How should this be manually tested?

1. Clone the repo 
2. Cd into that repo
3. run  `npm install`
4. After that run `npm run dev`

## What are the relevant pivotal tracker/Trello stories?

1. #184752075
2. #184752076
3. #184752078
